mlir/lib/Target/LLVMIR/ModuleTranslation.cpp

```diff
@@ -1351,6 +1351,18 @@ LogicalResult ModuleTranslation::convertOneFunction(LLVMFuncOp func) {
     llvmFunc->addFnAttr("no-signed-zeros-fp-math",
                         llvm::toStringRef(*noSignedZerosFpMath));
 
+  if (auto DenormalFPMath = func.getDenormalFpMath())
```

Comment:
```suggestion
  if (auto denormalFPMath = func.getDenormalFpMath())
```
nit: the variable names should start lowercase (below as well).

Reply:
Thanks, modified. The diff please see: [diff](https://github.com/llvm/llvm-project/compare/94bd71f97c6e35fac774611554d0069cc0e16836..c296c22b72f24c37acf37614b3f5001f1eb6475d#diff-0e9fd6478c7a83100254a3b40e2d97649a8d41c84506e1da7f8d281fc633b1f5R1354)